atd-vze/src/views/Fatalities/Fatalities.js

```diff
@@ -14,18 +15,22 @@ let queryConf = {
   },
   table: "view_fatalities",
   single_item: "crashes",
-  showDateRange: false,
+  showDateRange: true,
   columns: fatalityGridTableColumns,
   order_by: {},
   where: {},
   limit: 25,
   offset: 0,
 };
 
+const minDate = subYears(new Date(), 10);
```

Comment:
TIL I didn't realize that GridTable was extended with this `minDate` prop since the last time that I saw it. Very cool.